Sources/Shared/UI/Social/MessageComposerViewController.m

```diff
@@ -416,6 +423,18 @@ - (IBAction)onBtnSend:(id)sender
     
 }
 
+- (void)sendImageInBackgroundForDirectory:(NSString *)directory data:(NSData *)imageData
+{
+    if (selectedSpace){
+        NSString * driverName = [selectedSpace.groupId stringByReplacingOccurrencesOfString:@"/" withString:@"."];
+        [[FilesProxy sharedInstance] uploadFile:imageData asFileName:[directory lastPathComponent] inFolder:@"Mobile" ofDrive:driverName];
+        
+    } else {
+        [[FilesProxy sharedInstance] uploadFile:imageData asFileName:[directory lastPathComponent] inFolder:@"Public/Mobile" ofDrive:@"Personal Documents"];
```

Comment:
`Personal Documents` should be defined as 
`#define MOBILE_UPLOAD_PERSONAL_DRIVE @"Personal Documents"`